stdlib/http/src/main/ballerina/http/resiliency/http_load_balancer.bal

```diff
@@ -298,47 +296,30 @@ function performLoadBalanceAction(LoadBalancerActions lb, string path, Request r
     }
 
     while (loadBalanceTermination < lengthof lb.loadBalanceClientsArray) {
-        CallerActions loadBalanceClient = roundRobin(lb, lb.loadBalanceClientsArray);
-
-        match invokeEndpoint(path, request, requestAction, loadBalanceClient) {
-            Response inResponse => return inResponse;
-
-            error httpActionErr => {
-                if (!lb.failover) {
-                    return httpActionErr;
-                } else {
-                    loadBlancerInRequest = createFailoverRequest(loadBlancerInRequest, requestEntity);
-                    loadBalanceActionError.httpActionErr[lb.nextIndex] = httpActionErr;
-                    loadBalanceTermination = loadBalanceTermination + 1;
+        match lb.lbRule.getNextCallerActions(lb.loadBalanceClientsArray) {
+            CallerActions loadBalanceClient => {
+                match invokeEndpoint(path, request, requestAction, loadBalanceClient) {
+                    Response inResponse => return inResponse;
+                    error httpActionErr => {
+                        if (!lb.failover) {
```

Comment:
Is there any possibility of switch the if-else without using '!' operator?

Reply:
Yes, this can be done.